### ISSUE TITLE
Using phpcbf with stdin outputs the changed file to stdout instead of the summary report

### ISSUE THREAD

**Issue by valeryan** [squizlabs/PHP_CodeSniffer#2309]
Hello, I assume the title is the expected behavior of phpcbf when fixing php in memory but I would like to be able to retrieve the summary report as well. Is there a way to get the normal report that I would get if just passing a file path and the fixed code as the stdout? I am using phpcbf in my vscode extension and I am trying to switch from working with a file on disk to working in memory to address https://github.com/valeryan/vscode-phpcbf/issues/8 but my logic to handle the end of the process is somewhat based on the report output. 

Thanks

**Comment by gsherwood** [comment]
No, there is no way to have PHPCBF output both a report and the fixed file content at the same time.

I've worked on a PHPCBF plugin for an editor before, but did it in a slightly different way. Instead of using PHPCBF, I used PHPCS and asked for a `diff` report (and a full report). The full report was used to show what changed would be made, and the diff was used to make the changes. This helped fix file content that wasn't yet saved.

Not sure if that helps.
